crates/bevy_window/src/monitor.rs

```diff
@@ -0,0 +1,63 @@
+use bevy_ecs::component::Component;
+use bevy_ecs::prelude::ReflectComponent;
+use bevy_math::{IVec2, UVec2};
+use bevy_reflect::Reflect;
+
+#[cfg(feature = "serialize")]
+use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
+
+/// Represents a monitor attached to the system, which can be used to create windows.
```

Comment:
These docs should call out that, like windows, each monitor is associated with its own entity.